src/py42/clients/_archive_access.py

```diff
@@ -17,9 +17,9 @@ class FileType(object):
 
 
 class ArchiveAccessorManager(object):
-    def __init__(self, archive_client, storage_client_factory):
-        self._archive_client = archive_client
-        self._storage_client_factory = storage_client_factory
+    def __init__(self, archive_service, storage_service_factory):
+        self._archive_service = archive_service
```

Comment:
a lot the changes in these files are just around changing module -> client, client -> service, etc.

Reply:
Is there basis for this convention? All three of the words `module`, `client`, and `service` can easily mean the same thing.

Glad to get rid of the use of `module` though, because of python modules

Reply:
i kinda explained my reasoning in the description of the PR:
client = thing user interfaces with to make requests
service = web service, microservice, etc